### PR TITLE
up coverage to 100%

### PR DESCRIPTION
**Pull request by Pompeu**

### CODE REVIEW

File: test/spec/directives/numbers-only.spec.js [hunk 1]
```diff
@@ -68,5 +68,37 @@ describe('numbersOnly', function() {
         expect(element.val()).toBe('53.51');
       });
     });
+
+    describe('When is a number no has decimal values', function() {
```
Comment (masterl) [inline test/spec/directives/numbers-only.spec.js]: `When input number is an integer` or `When input number doesn't have decimal values` would be better

File: test/spec/directives/numbers-only.spec.js [hunk 2]
```diff
@@ -68,5 +68,37 @@ describe('numbersOnly', function() {
...
+      beforeEach(function() {
+        element = compileElement();
+        window.event = {keyCode: 49};
+        spyOn(String, 'fromCharCode').and.returnValue('0');
+        ngModel.$setViewValue('53');
+        element.triggerHandler('keypress');
+        rootScope.$digest();
+      });
+
+      it('should show numbers without decimal values', function() {
```
Comment (masterl) [inline test/spec/directives/numbers-only.spec.js]: should show **number** not **numberS**, there's only one number.

File: test/spec/directives/numbers-only.spec.js [hunk 3]
```diff
@@ -68,5 +68,37 @@ describe('numbersOnly', function() {
...
+        expect(element.val()).toBe('53');
+      });
+    });
+
+    describe('When has not a number if input', function() {
```
Comment (masterl) [inline test/spec/directives/numbers-only.spec.js]: `When input is empty`

**Comment by willmendesneto** [comment]
Closed and merged in https://github.com/willmendesneto/ng-numbers-only/commit/3c02410adec1afb8d71e9a0bb5a3d7ae903b2a76

Thanks @Pompeu @masterl

**Pull request merged**